book/src/11_advanced_component.md

```diff
@@ -124,8 +124,8 @@ impl<'a> System<'a> for Render {
     );
     
     fn run(&mut self, (active_cam, camera, transform, mesh) : Self::SystemData) {
-        let camera = camera.get(active_cam.0).unwrap();
-        let view_matrix = transform.get(active_cam.0).unwrap().invert();
```

Comment:
I don't understand this change either.

Reply:
Umm, that must be rebase doing something wonky, hold on.